torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -404,6 +406,24 @@ RegisterOperators reg(
              return 0;
            };
          }),
+     Operator(
+         "aten::save(t item, str filename) -> ()",
+         [](Stack& stack) {
+           auto filename = pop(stack).toStringRef();
+           auto value = pop(stack);
+
+           // Pickle the tensor
+           Pickler p;
+           p.start();
```

Comment:
`start` could be in the constructor but `finish` puts some necessary opcodes at the end of the binary blob so it needs to run before the stack is stored somewhere (so it can't be in the destructor). Because of that it's more clear I think to have both `start` and `finish`